addons/mail/tests/template_statusbar_time_testing.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+from unittest.mock import patch
+
+from odoo.addons.mail.tests.common import MockStatusbarTime
+
+
+class TestStatusbarTimeMixin(MockStatusbarTime):
```

Comment:
Would gladly have a "generic" test in ``test_mail`` addon: you can define a new test model inheriting from the mixin, and check if works on it. There are effectively tests for crm, project, helpdesk, but it is always a good idea to have "generic" (business-independent) tests. And it allows to modify and test your mixin without relying on a functional addon :) .